### PR TITLE
updating squads to include TestMCGRecovery 

### PR DESCRIPTION
**Pull request by vavuthu**
Signed-off-by: vavuthu <vavuthu@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/constants.py [hunk 1]
```diff
@@ -1338,7 +1338,7 @@
     "Brown": ["/z_cluster/"],
     "Green": ["/pv_services/", "/storageclass/"],
     "Blue": ["/monitoring/"],
-    "Red": ["/mcg/", "/rgw/"],
+    "Red": ["/mcg/", "/rgw/", "/system/"],
```
Comment (PrasadDesala) [inline ocs_ci/ocs/constants.py:1341]: @vavuthu E2E came up with system tests recently from 4.9. The responsibility of analysing this test failure belongs to Magenta squad. Please move it under magenta

Comment (vavuthu) [reply ocs_ci/ocs/constants.py]: done

**Comment by vavuthu** [comment]
verified locally:
```
Summary

1 tests ran in 64.73 seconds.
0 passed, 0 skipped, 1 failed, 0 errors, 0 expec ted failures, 0 unexpected passes
Squad Analysis - please analyze:
Failures:
Magenta squad

    tests/e2e/system/test_mcg_recovery.py::TestMCGRecovery::test_mcg_db_backup_recovery

Report generated on 10-Jan-2022 at 16:17:37 
```

File: ocs_ci/ocs/constants.py (hunk 1 above)
Comment (vavuthu) [reply ocs_ci/ocs/constants.py]: @PrasadDesala coud you update the "Tests Ownership (squads)" spreadsheet

Comment (PrasadDesala) [reply ocs_ci/ocs/constants.py]: Done

**Review by PrasadDesala — APPROVED** [review]

**Review by Akarsha-rai — APPROVED** [review]

**Pull request merged**